testing/chaosmonkey/batch.tf

```diff
@@ -40,8 +40,11 @@ resource "aws_batch_compute_environment" "batch_ce" {
 
     type = "EC2"
 
-    tags = var.tags
-  }
+    tags = merge(
+    var.tags,
+    {
+      "Name" = "${local.name_prefix}-asg"
+    },
```

Comment:
@zakqavalley Can you remove the extra , and close the merge function

Reply:
AS Benson said.
Also should that suffix be "-batch" rather than "-asg"?